core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -222,6 +222,7 @@ private[spark] object UIUtils extends Logging {
     <script src={prependBaseUri(request, "/static/timeline-view.js")}></script>
     <script src={prependBaseUri(request, "/static/log-view.js")}></script>
     <script src={prependBaseUri(request, "/static/webui.js")}></script>
+    <script src={prependBaseUri(request, "/static/scroll-button.js")} type="module"></script>
     <script>setUIRoot('{UIUtils.uiRoot(request)}')</script>
```

Comment:
![image](https://github.com/user-attachments/assets/ba655740-f029-4543-bfe4-3c57a9b8cc76)

Interesting. this `</script>` is red..

Reply:
This is a GitHub rendering issue